package.json

```diff
@@ -399,6 +399,11 @@
           "type": "boolean",
           "default": true,
           "description": "Specifes whether OmniSharp should use VS Code editor settings for C# code formatting (use of tabs, indentation size)."
+        },
+        "omnisharp.hugeProjectLimit": {
```

Comment:
This probably isn't a good name for a user-facing Option. In addition, because the option controls the extension and not OmniSharp, it should be prefixed with `csharp` rather than `omnisharp`. Maybe something like `csharp.maxProjectFileCountForDiagnosticAnalysis`?